content/06.methods.md

```diff
@@ -243,15 +243,16 @@ We aimed to define focal copy number units to avoid calling adjacent genes in th
 To determine the most focal units, we first considered the dominant status calls at the chromosome arm-level.
 If the chromosome arm dominant status was callable but not clearly defined as a gain or loss, we instead included the cytoband-level status call.
 Similarly, if a cytoband dominant status call was callable but not clearly defined as a gain or loss, we instead included gene-level status call.
-To obtain the gene-level data, we used the `mergeByOverlaps` [function](https://www.rdocumentation.org/packages/IRanges/versions/2.6.1/topics/findOverlaps-methods) from the IRanges package in R [@doi:10.1371/journal.pcbi.1003118] to find overlaps between the segments in the consensus CNV file and the exons in the GENCODE v27 annotation file (See **Key Resources Table**)	.
+To obtain the gene-level data, we used the `mergeByOverlaps` [function](https://www.rdocumentation.org/packages/IRanges/versions/2.6.1/topics/findOverlaps-methods) from the `IRanges` package in R [@doi:10.1371/journal.pcbi.1003118] to find overlaps between the segments in the consensus CNV file and the exons in the GENCODE v27 annotation file (See **Key Resources Table**)	.
```

Comment:
I would remove this link, and simply say we used the `IRanges` package in R. No need to reference the specific function.